$NetBSD: patch-aa,v 1.1.1.1 2010/02/18 02:56:35 seb Exp $

Use prefered form, warning free with current Perl, for require's argument.

--- Glue.pm.orig	2002-09-15 07:48:11.000000000 +0200
+++ Glue.pm
@@ -231,7 +231,7 @@ statements that work even though they lo
 use strict;
 use warnings;
 
-require 5.6.0;
+require 5.006_000;
 
 ######################################################################
 
